index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link 
+  rel="stylesheet" 
+  href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link 
+  href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  rel="stylesheet">
+  <link rel="stylesheet" href="./styles/home.css">
+
+  <title>Africa Tech Report</title>
+</head>
+<body>
+  <header class="header">
+    <div class="brand-logo">Brand</div>
+    <button type="button" aria-label="Mobile navigation toggle button" class="nav-toggle">
+      <span aria-hidden="true" class="hamburger"></span>
+    </button>
+    <nav class="nav">
+      <ul class="nav-list">
+        <li class="nav-list-item"><a href="./pages/about.html" class="nav-list-link link">About</a></li>
+        <li class="nav-list-item"><a href="#program-section" class="nav-list-link link">Program</a></li>
+        <li class="nav-list-item"><a href="#join" class="nav-list-link link">Join</a></li>
+        <li class="nav-list-item"><a href="./pages/about.html#partner-section" class="nav-list-link link">Partners</a></li>
+        <li class="nav-list-item"><a href="#" class="nav-list-link link">News</a></li>
+        <li class="nav-list-item"><a href="#" class="nav-list-link link">AT Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main">
+    <section class="headline-section">
+      <h1 class="headline-title">Africa Tech Report</h1>
+      <p class="headline-description">
+        Thanks to digital technologies, more people 
+        — no matter their income level or where they live — 
+        can access unprecedented amounts of information, take online jobs, enroll in e-courses, 
+        and receive life-saving care through telemedicine.
+      </p>
+    </section>
+    <section class="program-section" id="program-section">
+      <h2 class="program-section-title">Main Program</h2>
```

Comment:
- [ ] There are some differences in the horizontal and vertical placement of the `Main Program` sections between your and original design. Take a look to compare and fix it, please. 👇 
- Original design:
- 
![Screenshot (163)](https://user-images.githubusercontent.com/67087664/152133533-68250c18-1a58-42ef-bf69-f589d3046f16.png)
- Your design:
- 
![Screenshot (164)](https://user-images.githubusercontent.com/67087664/152133579-777dd277-558d-4065-89a5-59bcde0791b7.png)
Please remember this rule is for only desktop size you can leave your mobile size as it was before.